sound/soc/sof/intel/hda-loader.c

```diff
@@ -372,3 +372,148 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	dev_err(sdev->dev, "error: load fw failed err: %d\n", ret);
 	return ret;
 }
+
+// skl/kbl enable core and code loader DMA has some difference with apl/cnl
+// add the APIs for skl/kbl
+static int cl_stream_prepare_skl(struct snd_sof_dev *sdev, unsigned int format,
+				 unsigned int size, struct snd_dma_buffer *dmab,
+				 int direction)
+{
+	u32 tag = 0;
+	return tag;
+}
+
+static int cl_dsp_init_skl(struct snd_sof_dev *sdev, const void *fwdata,
+			   u32 fwsize)
```

Comment:
Btw, github can skew the formatting a little, so it's best to *always* run checkpatch.

Reply:
I have run the checkpatch, but don't know why here is still not aligned.

Reply:
this is just github rendering, if checkpatch is OK then you are good.